# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/mtransport/ipc/WebrtcTCPSocketCallback.h
# Commit: c43589fd0f90
# Full Hash: c43589fd0f90d174eb185b563406fd0a2e6ea712
# Author: Byron Campen [:bwc] <docfaraday@gmail.com>
# Date: 2019-09-18 21:49:27
# Regressor Bug: 1569183
# File Overlap Count: 2
# Description:
#   Bug 1569183: Update OnConnected callbacks to specify what type of proxying (if any) is being used, and implement IsProxied based on this information. r=mjf
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D45186
# ==============================================================================

diff -r 940d7f787bea -r c43589fd0f90 media/mtransport/ipc/WebrtcTCPSocketCallback.h
--- a/media/mtransport/ipc/WebrtcTCPSocketCallback.h	Tue Sep 17 18:18:15 2019 +0000
+++ b/media/mtransport/ipc/WebrtcTCPSocketCallback.h	Tue Sep 17 18:18:33 2019 +0000
@@ -17,7 +17,7 @@
   NS_INLINE_DECL_PURE_VIRTUAL_REFCOUNTING
 
   virtual void OnClose(nsresult aReason) = 0;
-  virtual void OnConnected() = 0;
+  virtual void OnConnected(const nsCString& aProxyType) = 0;
   virtual void OnRead(nsTArray<uint8_t>&& aReadData) = 0;
 
  protected: